vnext/Microsoft.ReactNative.IntegrationTests/pch.h

```diff
@@ -2,9 +2,9 @@
 
 #define NOMINMAX
 
+#include <functional/functor.h>
```

Comment:
@vmoroz could you in the future try to avoid these types of changes in PRs marked to be backported? Clean up in these PRs often means merge conflicts because other things have touched this since 0.62.